k-distribution/src/main/assembly/bin.xml

```diff
@@ -106,5 +106,11 @@
       <fileMode>775</fileMode>
       <directoryMode>775</directoryMode>
     </fileSet>
+    <fileSet>
+      <directory>${project.basedir}/../hs-backend-booster/target/stack-install</directory>
+      <outputDirectory>/bin</outputDirectory>
+      <fileMode>775</fileMode>
+      <directoryMode>775</directoryMode>
+    </fileSet>
```

Comment:
Hey @goodlyrottenapple, besides these binaries, is there anything that should be included in the package? E.g., the old Haskell Backend includes `kore`, but I couldn't find anything correspondent to the old `kore` in the booster repo...

Reply:
Thanks!